scripts/sync/support.rb

```diff
@@ -93,6 +93,33 @@ def absolutize_links(repository_slug, path_transformer, contents_source_path, so
     @lines = lines.collect { | line | UrlAbsolutizer.absolutize_links(line, repository_slug, path_transformer, contents_source_path, source_paths) }
   end
 
+  def escape_things_that_look_like_jsx_tags
+    in_code_block = false
+    @lines = lines.collect do | line |
+      in_code_block = !in_code_block if line.start_with?('```')
```

Comment:
Is it starts with or just contains?

Reply:
It should really be "or or more spaces, then three backticks". I've just been coding for the markdown we have, rather than making it work for all markdown.